cmc/values.yaml

```diff
@@ -0,0 +1,236 @@
+tags:
+  cmc-claim-store-postgresql-pod: false # claim-store chart
+  cmc-citizen-frontend-pod: false       # claim-store chart
+  cmc-ccd-stack: true                   # claim-store chart
+  idam-pr: false                        # manually configure whitelists in IDAM until IDAM here
+  ccd-idam-pr: false                    # manually configure whitelists in IDAM until IDAM here
+  rpe-feature-toggle-api-pod: true      # citizen-frontend dep
+  postgresql-pod: false                 # any chart specific postgres
+  blobstorage: false                    # blob storage for dm-store
+
+### CMC SERVICES ###
+cmc-claim-store:
+  java:
+    image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest
+    readinessDelay: 180
+    livenessDelay: 180
+    readinessPeriod: 60
+    livenessPeriod: 60
+    keyVaults:
+    environment:
+      CLAIM_STORE_DB_USERNAME: hmcts
+      CLAIM_STORE_DB_PASSWORD: hmcts
+      CMC_DB_USERNAME: hmcts
+      CMC_DB_PASSWORD: hmcts
+  ccd:
+    readinessDelay: 180
+    livenessDelay: 180
+    readinessPeriod: 60
+    livenessPeriod: 60
+    adminWeb:
+      enabled: true
+      environment:
+        NODE_TLS_REJECT_UNAUTHORIZED: 0
+    caseManagementWeb:
+      enabled: true
+      environment:
+        NODE_TLS_REJECT_UNAUTHORIZED: 0
```

Comment:
and this?

Reply:
no cert and communicate on https to api-gateway - it didnt work without, maybe changing protocol but im sure i looked at this (quickly)